Tasks/ExtractFiles/extractFiles.ts

```diff
@@ -0,0 +1,307 @@
+/// <reference path="../../definitions/vsts-task-lib.d.ts" />
```

Comment:
All uses of path.sep in this file need to account for either '/' or '\' being legal on Windows.  path.sep only returns '\' on Windows.